Source/Applications/SystemCenterNotification/Scripts/TSX/EmailTypes/Subscriptions.tsx

```diff
@@ -132,43 +134,48 @@ const Subscriptions = (props: IProps) => {
+                                            HeaderStyle={{ width: 'auto' }}
+                                            RowStyle={{ width: 'auto' }}
                                         > Last Name
                                         </Column>
                                         <Column<SubscribeEmails>
                                             Key={'Email'}
                                             AllowSort={true}
                                             Field={'Email'}
-                                            HeaderStyle={{ width: props.Record.RequireApproval ? '20%' : '25%' }}
-                                            RowStyle={{ width: props.Record.RequireApproval ? '20%' : '25%' }}
+                                            HeaderStyle={{ width: 'auto' }}
+                                            RowStyle={{ width: 'auto' }}
                                         > Email
                                         </Column>
                                         <Column<SubscribeEmails>
                                             Key={'AssetGroup'}
                                             AllowSort={true}
                                             Field={'AssetGroup'}
-                                            HeaderStyle={{ width: props.Record.RequireApproval ? '20%' : '25%' }}
-                                            RowStyle={{ width: props.Record.RequireApproval ? '20%' : '25%' }}
+                                            HeaderStyle={{ width: 'auto' }}
+                                            RowStyle={{ width: 'auto' }}
                                         > Assets
                                         </Column>
                                         {props.Record.RequireApproval ?
                                             <Column<SubscribeEmails>
                                                 Key={'Approved'}
                                                 AllowSort={true}
                                                 Field={'Approved'}
-                                                HeaderStyle={{ width: '20%' }}
-                                                RowStyle={{ width: '20%' }}
+                                                HeaderStyle={{ width: 'auto' }}
+                                                RowStyle={{ width: 'auto' }}
                                                 Content={({ item }) => item.Approved ? <ReactIcons.CheckMark Color="var(--success)" /> :
-                                                    <button type="button" className="btn btn-primary btn-sm" onClick={() => approve(item)}>Approve</button> }
+                                                    <button className="btn btn-sm"
+                                                        data-tooltip='approve'
+                                                        onMouseEnter={() => setHover('approve')}
```

Comment:
This is gonna lead to bugs if there is more than one row. Datatooltip needs to include the key of the row.